src/tpm2_getmanufec.c

```diff
@@ -206,9 +206,20 @@ int createEKHandle()
     creationPCR.count = 0;
 
     /*To Create EK*/
-    rval = Tss2_Sys_CreatePrimary(sysContext, TPM_RH_ENDORSEMENT, &sessionsData, &inSensitive, &inPublic,
-                                                                &outsideInfo, &creationPCR, &handle2048ek, &outPublic, &creationData, &creationHash,
-                                                                &creationTicket, &name, &sessionsDataOut);
+    rval = Tss2_Sys_CreatePrimary(sapi_context, 
+                                  TPM_RH_ENDORSEMENT, 
+                                  &sessionsData, 
+                                  &inSensitive, 
+                                  &inPublic,
+                                  &outsideInfo, 
+                                  &creationPCR, 
+                                  &handle2048ek, 
+                                  &outPublic, 
+                                  &creationData, 
+                                  &creationHash,
+                                  &creationTicket, 
+                                  &name, 
+                                  &sessionsDataOut);
```

Comment:
line's should be split to nearest whitespace at 80 characters per the coding standard